usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -514,12 +514,16 @@ def get_or_create_object_class(row_object_class, row_direct_reimbursable, logger
     return obj_class
 
 
-def get_or_create_program_activity(program_activity_code):
+def get_or_create_program_activity(row, submission_attributes):
     # We do it this way rather than .get_or_create because we do not want to
     # duplicate existing pk's with null values
-    prg_activity = RefProgramActivity.objects.filter(ref_program_activity_id=program_activity_code).first()
-    if prg_activity is None and program_activity_code is not None:
-        prg_activity = RefProgramActivity.objects.create(ref_program_activity_id=program_activity_code)
```

Comment:
I'm actually really puzzled here.

It *looks like* row['program_activity_code'] ought to be used to populate `RefProgramActivity.program_activity_code`, both because of the matching name, and because the model is defined with

`        unique_together = (('program_activity_code', 'budget_year', 'responsible_agency_id', 'allocation_transfer_agency_id', 'main_account_code'),)`

But, all this time, we've been using it to populate `ref_program_activity_id`.  

So what is `ref_program_activity_id` for, and why did we define it as an `IntegerField` rather than an `AutoField` (like most of our surrogate primary keys are)?